.github/workflows/nuget.yml

```diff
@@ -0,0 +1,69 @@
+name: Package nuget
+on:
+  push:
+    branches: [ "main", "feature/gh-nuget-workflow" ]
+  pull_request:
+    branches: [ "main" ]
+env:
+  RESULT_DIR: result
+jobs:
+  build_test_publish:
+    runs-on: ubuntu-latest
+    environment: nuget
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v3
+        with:
+          global-json-file: global.json
+          source-url: "https://nuget.pkg.github.com/${{ github.repository_owner }}/index.json"
+        env:
+          NUGET_AUTH_TOKEN: ${{secrets.GITHUB_TOKEN}}
+      - name: Use Node.js 18
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+      - name: Local nuget source
+        run: |
+          mkdir -p result 
+          dotnet nuget add source \
+            --name local \
+            "./$RESULT_DIR"
+          dotnet nuget list source
+      - name: Restore python dependencies
```

Comment:
I think we can remove this step. Because we don't run the tests, we don't need python.